helpers/addresses.py

```diff
@@ -143,6 +143,13 @@
             "ops_harvester": "0x73433896620E71f7b1C72405b8D2898e951Ca4d5",
             "ops_external_harvester": "0x64E2286148Fbeba8BEb4613Ede74bAc7646B2A2B",
         },
+        "treasury_councillors": {
+            "councillor1": "0xc87e98867B392BDBA50FfCfE53014226FE1fAaE7",
+            "councillor2": "0xae3FB0474841D69AAE9446108F41D01f11E8BeF4",
+            "councillor3": "0xd1911EEDf284BCD3559de6A06ef67233023e0dFe",
+            "councillor4": "0x33Dd3EC6CCB2736dc285bde70aADe5C45bFaD588",
+            "councillor5": "0x3f993093c3917Fe7A117f37cFD008775249f59c8",
```

Comment:
Just curious, `councillor1/2/3/4` address looks pretty new and with zero balance (except `councillor1` receiving a transfer from CoinBase a few days ago), are they created in advance out of `create2` ?

Reply:
Correct, councillors created new accounts in most cases for this purpose. They are all confirmed and the transaction will be ultimately reviewed and confirmed by themselves.